docs/pages/versions.js

```diff
@@ -12,12 +13,13 @@ export default function Page() {
 }
 
 async function getBranches() {
+  const githubAuthorizationToken = process.env.GITHUB_AUTH || '';
+
   const result = await fetch('https://api.github.com/repos/mui-org/material-ui-docs/branches', {
     headers: {
-      Authorization: `Basic ${Buffer.from(process.env.GITHUB_AUTH).toString('base64')}`,
```

Comment:
Threw locally if this wasn't set (Buffer.from does not accept undefined). Seems like this causes the API to gracefully fallback to no-token API queries.